pandas/core/internals/blocks.py

```diff
@@ -2247,7 +2247,7 @@ class ObjectBlock(Block):
     _can_hold_na = True
 
     def _maybe_coerce_values(self, values):
-        if issubclass(values.dtype.type, (str, bytes)):
+        if issubclass(values.dtype.type, str):
```

Comment:
why are you changing this?

Reply:
I have added this in #39065, which caused the regression. have to handle the replace issue at another place

Reply:
oh i